src/data/ipfs.js

```diff
@@ -308,5 +307,10 @@ async function uploadMetadataFile({
     metadata.contentRating = contentRating
   }
 
+  if (rights === 'custom') {
+    metadata.rightsUri = rightUri
```

Comment:
rightsUri or rightUri? I get confused :sweat_smile:

Reply:
> rightsUri or rightUri? I get confused 😅

Hehe good point, we should match them all, on the indexer it's using what for now?

Reply:
The indexer uses `rightUri`